### DISCUSSION TITLE
Some RFCs are missing from the RFC's overview: https://github.com/fsharp/fslang-design/blob/master/RFCs

### DISCUSSION THREAD

**Discussion by abelbraaksma** [fsharp/fslang-design#468] [General]
I'm not sure how/when RFCs are being moved there, but I just noticed that a bunch of RFC's are missing in the overview (https://github.com/fsharp/fslang-design/blob/master/RFCs), leading to link-rot (see: https://github.com/dotnet/fsharp/pull/7839, the OP had a link to the then-correct location, it's now only available on the "preview" branch, as found by @Happypig375's [comment](https://github.com/dotnet/fsharp/pull/7839#issuecomment-635893218)).

I think we should strive to be consistent/complete here and make sure all RFC's are listed there (there are some other missing numbers besides 1080). Or, if there are different collections and they ought to be moved, prevent link-rot by allowing redirects or something?

I'm not sure of the policy here, just saying I often hit a 404 when clicking on an RFC link in some issue, which isn't good for posterity.

Edit: I think I found the cause: https://github.com/fsharp/fslang-design/commit/c7109451bf4a8436c31f017fbde734d699921c7d (cc: @cartermp). I thought when moving stuff, that Github would update references, but apparently it doesn't. Do we have a better option?

**Comment by cartermp** [comment]
No, github doesn't auto-redirect. RFCs are moving to folders that represent their status in F# itself.

**Reply by abelbraaksma** [reply]
This old discussion can be closed. But I don’t seem to have the option to mark this as the answer, or close the thread for that matter.

**Reply by cartermp** [reply]
I don't think there's a way to close a discussion (or at least I lack permissions to do so).

**Reply by vzarytovskii** [reply]
I think, the only way is to convert it to issue and close it. Or at least it's the only one I've found.

**Reply by abelbraaksma** [reply]
Apparently, locking is possible / should be possible by the owner: https://github.com/community/community/discussions/28745?sort=top, but I don’t see that option here. 🤔 

Maybe having that available or not is a setting on the repo. Not having it means discussions stay open forever. In other repos, I do see these options. Ah well.

**Comment by abelbraaksma** [comment]
Close button is back, possibly because I have (slightly) more rights now lol (but it looks like it is still possible to comment after it is closed… ah well…)
